tests/playbooks/all.yml

```diff
@@ -1,4 +1,5 @@
-- import_playbook: e2e.yaml
+- import_playbook: kubevirt_vm.yml
```

Comment:
Shouldn't we keep this in alphabetical order?

Reply:
I though we'd start with the more specific per–module tests first and if those pass, leave `e2e.yaml` for a more complex mix of everything.